ray-operator/controllers/ray/utils/validation_test.go

```diff
@@ -156,6 +163,27 @@ func TestValidateRayClusterSpecGcsFaultToleranceOptions(t *testing.T) {
 			expectError:  true,
 			errorMessage: errorMessageRedisAddressSet,
 		},
+		{
```

Comment:
These two tests have already been tested in `TestValidateRayClusterSpecRedisPassword`.